packages/components/src/public/QueryColumn.ts

```diff
@@ -286,6 +286,13 @@ export class QueryColumn extends Record({
         return fieldKey;
     }
 
+    getLookupKey(): string {
```

Comment:
I think we should make this `get lookupKey(): string {` so it can be used as an attribute, like `isFileInput` below.

Reply:
Done.